engine/hooks/kafka.go

```diff
@@ -0,0 +1,157 @@
+package hooks
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/fsamin/go-dump"
+	"gopkg.in/bsm/sarama-cluster.v2"
+
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+func (s *Service) saveKafkaExecution(t *sdk.Task, error string, nbError int64) {
+	exec := &sdk.TaskExecution{
+		Timestamp:           time.Now().UnixNano(),
+		Type:                t.Type,
+		UUID:                t.UUID,
+		Config:              t.Config,
+		Status:              TaskExecutionDone,
+		LastError:           error,
+		NbErrors:            nbError,
+		ProcessingTimestamp: time.Now().UnixNano(),
+	}
+	s.Dao.SaveTaskExecution(exec)
+}
+
+func (s *Service) startKafkaHook(t *sdk.Task) error {
+	var kafkaPlatform, kafkaUser, projectKey, consumerGroup, topic string
+	for k, v := range t.Config {
+		switch k {
+		case "platform":
+			kafkaPlatform = v.Value
+		case "consumer group":
```

Comment:
const for theses kafka keys could be great "consumer group" / "platform" / "topic"